model-engine/model_engine_server/core/celery/app.py

```diff
@@ -504,7 +504,7 @@ def _get_backend_url_and_conf(
     elif backend_protocol == "s3":
         backend_url = "s3://"
         if aws_role is None:
-            aws_profile = os.getenv("AWS_PROFILE", infra_config().profile_ml_worker)
+            aws_profile = os.getenv("S3_WRITE_AWS_PROFILE", infra_config().profile_ml_worker)
```

Comment:
do we have `S3_WRITE_AWS_PROFILE` inside the celery forwarder? just want to make sure that this works for (gateway, endpoint builder, celery forwarder, batch job orchestration pod)

Reply:
I guess this `s3_write_aws_profile` means a few things, e.g. in the gateway I think it has to do with some s3 repository for storing llm fine tune jobs, in celery workers it means where completed tasks get written, I think this is fine though since it's all related to s3

guess the perms for this are actually "read, write, maybe list" as opposed to just "write" as well,

Reply:
discussed offline. reverting back to `profile_ml_worker`